### DISCUSSION TITLE
How to anticipate whether smart cut will work or not ?

### DISCUSSION THREAD

**Discussion by KaKi87** [mifi/lossless-cut#2065] [Q&A]
Hi,

When cutting a video, here's what I'm currently doing :

- Add non-keyframe segments ;
- Export ;
- Turn non-working segments into keyframe segments ;
- Export.

Is there any way to anticipate where will smart cut work or not, in order not to waste time trying ?

Thanks

**Comment by hrxn** [comment]
Somewhat, although I wouldn't really bet serious $$ on it..

The more "standard" a file is (check with mediainfo or ffprobe), the more likely it is that smart cut will work.

Where "standard" means MP4 container without any unusual atoms, AVC/h264 encoded video stream (just one) without any exotic encoding settings etc.

**Comment by mifi** [comment]
Unfortunately there's no sure-fire way to know. It's such a complex operation with 100s of input parameters that affect how well it will work. Much discussion here #126
